rest/pom.xml

```diff
@@ -202,13 +202,12 @@
     </plugins>
   </build>
 
-  <profiles>    
+  <profiles>
      <profile>
         <id>auth</id>
         <activation>
             <property>
                 <name>auth</name>
```

Comment:
with this config, if we build without `-Dauth`, no auth will be set up. But if we use `-Dauth`, either **with or without value**, **rest will be built with auth**. So even if someone uses `-Dauth=false`, auth will be enabled.

@matejonnet is this ok for us?